src/Storages/FileLog/StorageFileLog.cpp

```diff
@@ -175,7 +175,7 @@ StorageFileLog::StorageFileLog(
     , path(path_)
     , metadata_base_path(std::filesystem::path(metadata_base_path_) / "metadata")
     , format_name(format_name_)
-    , log(getLogger("StorageFileLog (" + table_id_.table_name + ")"))
+    , log(getLogger("StorageFileLog (" + table_id_.database_name "." + table_id_.table_name + ")"))
```

Comment:
Here and in other places:
```suggestion
    , log(getLogger("StorageFileLog (" + table_id_.getNameForLogs() + ")"))
```